src/plugins/discover/public/application/view_components/utils/use_search.ts

```diff
@@ -50,6 +51,10 @@ export interface SearchData {
   bucketInterval?: TimechartHeaderBucketInterval | {};
   chartData?: Chart;
   title?: string;
+  statusBody?: {
```

Comment:
nit:

i still think maybe we maybe we can:

```
status: ResultStatus | DetailedResultStatus
```

and then the `DetailedResultStatus` is 

```
status: ResultStatus,
message: string,
statusCode?: number,
elapsedMs?: number,
```